examples/attendance/Entity Resolution on Organizations.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import bigbang.ietfattendance as ia\n",
+    "import bigbang.process as process\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import Levenshtein"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Start by getting the attendence information for IETF."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(1639, 6)\n"
+     ]
+    }
+   ],
+   "source": [
+    "attendance106 = ia.attendance_tables(106)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "What organizations are best represented?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Organization\n",
+       "                                 283\n",
+       "Google                            33\n",
+       "Cisco                             30\n",
+       "Nokia                             27\n",
+       "Cisco Systems                     27\n",
+       "Juniper Networks                  26\n",
+       "Ericsson                          18\n",
+       "Huawei                            18\n",
+       "Apple                             13\n",
+       "Internet Society                  13\n",
+       "Microsoft                         12\n",
+       "Facebook                          12\n",
+       "ICANN                             11\n",
+       "Huawei Technologies               10\n",
+       "NIST                               9\n",
+       "Akamai Technologies                9\n",
+       "RIPE NCC                           9\n",
+       "Futurewei                          8\n",
+       "NTT                                7\n",
+       "UPSA                               7\n",
+       "Verisign                           7\n",
+       "AMS                                7\n",
+       "ETRI                               7\n",
+       "MISA                               7\n",
+       "Mozilla                            7\n",
+       "Fastly                             6\n",
+       "Tsinghua University                6\n",
+       "ISC                                6\n",
+       "Huawei Technologies Co., Ltd.      6\n",
+       "Inria                              6\n",
+       "Name: First Name, dtype: int64"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "attendance106.groupby('Organization') \\\n",
+    "             .count()['First Name'] \\\n",
```

Comment:
I believe you can use `.size()` rather than `.count()['First Name']` here and it's clearer what's intended.